vmm/src/vm.rs

```diff
@@ -519,7 +518,7 @@ pub struct Vm {
     vm: Arc<dyn hypervisor::Vm>,
     #[cfg(all(feature = "kvm", target_arch = "x86_64"))]
     saved_clock: Option<hypervisor::ClockData>,
-    #[cfg(feature = "acpi")]
+    #[allow(dead_code)]
```

Comment:
Is it the way we want to handle this? The alternative would be to provide the features for which this is relevant.

Reply:
Yeah (more or less), as the clippy will also check `--features=kvm` without `acpi` on x86_64, and in this case this `numa_nodes: NumaNodes` field is never used. The `#[allow(dead_code)]` is intended just to mute clippy.

On arm everything is fine (both `--features=kvm` and `--features=kvm, acpi` will use this field).

It is kinda tricky to provide a proper gate here...